content/virtualization.md

```diff
@@ -0,0 +1,199 @@
+>
+> When trying the download a few more times, I either saw same window with random text, or I received a "This page isn't responding" prompt.
+>
+> If either of these happen when you download, right-click the link above and select "Save Link As..." and save the file to your Downloads folder.
+
+## Virtualiaztion Features
+
+Now that the VM is set up, and drivers are installed, we can now use the advanced features of GNOME Boxes.
+
+Most of these features can be found in the VM's settings menu. To reach the VM settings menu, click on your VM in the GNOME Boxes list to start the VM. You can now select the <kbd>&vellip;</kbd> button at the top right of GNOME Boxes, then select "Properties" to open the VM settings menu.
+
+### Copy and Paste
+
+Once your VM has the `spice-vdagent` or an equivelent package installed on your VM, reboot the VM once to load this driver. At this point, you will now be able to copy and paste between your VM and the Host just like copy and pasting between applications.
+
+### Share a USB Device
+
+![Devices and Shares](/images/gnome-boxes/boxes-vmprefs-devices_and_shares.png)
+
+One of the features of GNOME Boxes, is the ability to transfer a USB device connected to your Host to the VM. You can find this option in the VM settings menu, under the "Devices & Shares" tab. On this tab, under the "USB devices" heading, you will see a list of the USB devices found on your system. If you click the toggle bottom on the right, the USB device will disappear from your Host operating system, and it will appear inside your VM.
+
+### Share a Folder
+
+_For this guide, we used GNOME-based Linux distributions, so the VMs UI closely matched Pop!_OS and Ubuntu. Accessing shared folders with other desktop environments is possible, but you will need to refer to their documentation for the details_
+
+![Folder Shares](/images/gnome-boxes/boxes-folder-shares.png)
+
+To share a folder from the Host with your VM, you will need the `spice-webdavd` or an equivelent package installed on your VM. From the VM settings menu, select the "Devices and Shares" tab and scroll to the bottom. At the bottom of this menu, you will see the "Folder Shares" section. Click the <kbd>+</kbd> button to show the folder share pop-up.
+
+#### Selecting the Host's Folder
+
+In this pop-up, the "Local Folder" refers to the folder's name on the Host system. This defaults to the "Public" folder in your Home directory. If you click the drop down, you can quickly select some of the important folders on your system. If you do not wish to use one of these folders, select "Other..." This will open a folder selection window, where you can then navigate to the folder you wish to share with the VM. Once you are in the folder you wish to share, click "Open" in the top right corner.
```

Comment:
Minor formatting - This can stay as is for now, but going forward I've always preferred reading step-by-step instructions in step format. In paragraph form, this can be a lot for a reader to keep in their head, especially since there's no visual context. This breaks up the process, and gives the reader a reference point if they get stuck on a specific stage. Consider:

1. Click the drop down to quickly select important folders on your system.
2. If the folder you need isn't listed, click "Other" to navigate to the folder you want to share with the VM.
3. Once you have located the folder to share, click "Open" in the top right corner.